ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -11,6 +11,7 @@ Define {
 	SpeakerAmps1 "0"
 	HeadsetCodec1 ""
 	MicCodec1 ""
+	Dmics1 "0"
```

Comment:
Apparently, this string should be empty by default (Empty condition for include).

Reply:
the machine driver will set cfg-dmics=0, are you saying we should only conditionally set the component string when we have a non-zero value?

Reply:
we could also write 'mic:dmic' if that was desired, so that there's a built-in mutual exclusion in the driver.